server/src/main/java/com/sauron/fake/util/FakeBanksConstants.java

```diff
@@ -14,6 +14,7 @@
 	
 	public static final String CONTEXT_PATH = "/fake";
 	public static final String TRANSACTIONS_ENDPOINT = "/transactions";
+	public static final String CURRENT_BALANCE_ENDPOINT = "/balance";
```

Comment:
Better would be to do /current-balance. Endpoint /balance can mean balance in some time in past